tests/integration/tier1/changed-yum-conf/main.fmf

```diff
@@ -1,4 +1,17 @@
-summary: changed-yum-conf
+summary: Changed yum configuration file
+
+description: |
+    Test the scenario in which the user modifies /etc/yum.conf
+    before the conversion. In that case during the conversion
+    the config file does not get replaced with the config file
+    from the RHEL package (%config(noreplace)) and we need to
+    make sure that we patch the config file to get rid of the
+    distroverpkg config key. Leaving the distroverpkg key there
+    would lead to errors when calling yum after the conversion for not
+    expanding the $releasever variable properly.
```

Comment:
```suggestion
    Test the scenario in which the user modifies /etc/yum.conf
    before the conversion. In that case, during the conversion,
    the config file does not get replaced with the config file
    from the RHEL package. Make sure that C2R patch the config file
    and verify that the yum works after a successful conversion. 
```

Reply:
Keep it simple and describe only what is happening in the test. The details of implementation should not be present here.